src/sagemaker/workflow/steps.py

```diff
@@ -218,11 +249,12 @@ def __init__(
             name (str): The name of the transform step.
             transformer (Transformer): A `sagemaker.transformer.Transformer` instance.
             inputs (TransformInput): A `sagemaker.inputs.TransformInput` instance.
+            cache_config (CacheConfig):  An instance to enable caching.
```

Comment:
Maybe follow convention of other docstrings for such arguments? Something like:

```suggestion
            cache_config (CacheConfig):  A `sagemaker.workflow.steps.CacheConfig` instance for configuring caching.
```